src/config/index.js

```diff
@@ -0,0 +1,15 @@
+const env = process.env.NODE_ENV || 'development';
+
+const all = {
+  env,
+  passwordSalt: process.env.PASSWORD_SALT || 'salty',
+};
+
+try {
+  const override = require(`./env/${env}`); // eslint-disable-line
```

Comment:
Cette instruction lève une erreur en développement : 
voir-et-localiser-sso-0  | Cannot find module './env/development'

Reply:
Tu as testé en local ? Ça échoue ou bien ça affiche  "No specific configuration for ${env}"